### PR TITLE
Numbers Done

### PR DESCRIPTION
**Pull request by bbiehl**

### CODE REVIEW

File: lib/numbers.md [hunk 1]
```diff
@@ -5,20 +5,26 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
...
+A '+' with numbers will perform the addition operation.  For example, 2 + 7 => 9
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+Some mathematical operations can be performed on strings, some can not.  
+Strings can be added to other strings, i.e. "20" + "3" => "203".  Here we are simply adding the contents of the strings.
+Strings can be multiplied by a number, i.e. "20" * 3 => "202020". Here we are telling Ruby to add the contents of the string "20" + "20" + "20".
+Note: for this question the string must come first, i.e. 3 * "20" results in an error because we can not multiply a number by a string.
+All other mathematical operations with respect to strings will result in an error because they do not follow any logic.
```
Comment (sumeetjain) [inline lib/numbers.md:25]: So say we did have a string `"20"` and wanted to divide it by something. How would you make that work?

Comment (bbiehl) [inline lib/numbers.md:25]: Sorry, if we want to divide string "20" by something we can convert the string to an integer using .to_i

Comment (sumeetjain) [inline lib/numbers.md:25]: What if we wanted to divide it by 3? Or divide `"5"` by 2? Assuming we wanted the exact [quotient](https://www.mathsisfun.com/definitions/quotient.html), would we still use `.to_i`?

Comment (bbiehl) [inline lib/numbers.md:25]: If we wanted to divide by 3 or divide '5' by 2 we would want to us .to_f

**Comment by sumeetjain** [comment]
:+1:

**Pull request closed without merging**